### PR TITLE
Make scancode parallelism configurable

### LINKED ISSUE #609: Scancode parallelism is hardcoded to 2 processes
See here: https://github.com/clearlydefined/crawler/blob/0738316eaf5a9ef80f96cdaaeae38e2695688974/config/cdConfig.js#L98

Basically, scancode is always run at 2 parallel processes maximum, which becomes a bottleneck when running CD crawler on bigger machines.

The 2 makes sense as a default, since we see good CPU utilization in most of the current crawler deployments (which currently use at most 4 CPUs, as far as I know), but this number should be configurable for those special cases when larger instances are used for crawler.

### PR DESCRIPTION
**Pull request by RomanIakovlev**
Fixes #609 
Replaces #611

### CODE REVIEW

**Review by ljones140 — COMMENTED** [review]
Amazing work

**Review by ljones140 — APPROVED** [review]

File: config/cdConfig.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ module.exports = {
         // '--quiet'
       ],
       timeout: 1000,
-      processes: 2,
+      processes: Number(process.env.CRAWLER_SCANCODE_PARALLELISM) || 2,
```
Comment (elrayle) [inline config/cdConfig.js:98]: To allow this to be set using the normal configuration approach or from an env var, follow the pattern in... https://github.com/clearlydefined/crawler/blob/0738316eaf5a9ef80f96cdaaeae38e2695688974/ghcrawler/bin/www.js#L14

```suggestion
      processes: Number(config.get(CRAWLER_SCANCODE_PARALLELISM) || process.env.CRAWLER_SCANCODE_PARALLELISM) || 2,
```

**Review by elrayle — CHANGES_REQUESTED** [review]
One change requested, then this looks good.

**Review by elrayle — APPROVED** [review]
Thanks for making this configurable.  This will really help with long term maintenance and scalability as needed.

**Pull request merged**